scripts/ci/check_compliance.py

```diff
@@ -566,98 +579,81 @@ def check_no_undef_outside_kconfig(self, kconf):
 
 {undef_desc}""")
 
+    # Many of these are symbols used as examples. Note that the list is sorted
+    # alphabetically, and skips the CONFIG_ prefix.
+    UNDEF_KCONFIG_WHITELIST = {
+        "ALSO_MISSING",
+        "APP_LINK_WITH_",
+        "APP_LOG_LEVEL", # Application log level is not detected correctly as
+                         # the option is defined using a template, so it can't
+                         # be grepped
+        "ARMCLANG_STD_LIBC",  # The ARMCLANG_STD_LIBC is defined in the 
```

Comment:
> scripts/ci/check_compliance.py:590 Trailing whitespace (trailing-whitespace)

check_compliance seems to be very effective against itself :)

Reply:
🤣